### ISSUE TITLE
Add documentation comment on all public methods/property

### ISSUE THREAD

**Issue by neuecc** [Cysharp/R3#200]

**Comment by Antoshidza** [comment]
yes please, often I think that I don't know possibilities of R3 so I can't decide what to use to implement feature: search for already existing approach in R3 or invent myself
